### PR TITLE
Update Number Fields

### LINKED ISSUE #162: Update money fields with proper decimal stringency
Money fields in crux currently will accept no more than 2 decimal places, and that stringency needs to be noted in the docs for these fields (including on the import/export page).

So, the following fields need to somehow communicate the fact that **no more than 2 decimal places are allowed**:

**Orders** (as applicable): 

* shipping_cost
* price
* expected_sku_cost
* sku_cost
* estimated_shipping_cost
* drop_ship_fee
* per_order_fee
* estimated_tax

**Products** (as applicable):

* default_shipping_cost
* default_handling_cost
* msrp
* minimum_advertised_price
* cost
* shipping_cost
* handling_cost

### PR DESCRIPTION
**Pull request by nickcoleman**
Update the number fields in Product and Orders to reflect to be decimal (with 2 decimal places) and integer as appropriate.  

The issue specifically wanted money fields to be updated ... but, we did both decimal and integer while reviewing the docs.

fixes #162

### CODE REVIEW

File: _includes/product/response/item.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ uuid
 : (string) Universal Unique Identifier for the Item
 
 skus
-: (list) The Stock Keeping Units (SKUs) list contains individual SKUs, or Item-variants, with their SKU-level data
+: (array) The Stock Keeping Units (SKUs) contains individual SKUs, or Item-variants, with their SKU-level data
```
Comment (Gidgidonihah) [inline _includes/product/response/item.md]: "The Stock Keeping Units (SKUs) array contains..." or "The array contains..."

File: _includes/product/response/item.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ msrp_range
 : (object) The Manufacturer's Suggested Retail Price Range object contains the minimum and maximum MSRPs for the Item, based on the available variants (SKUs).
 
 product_images
-: (list) The Product Images list stores a list of images for the item, based on the available variants (SKUs)
+: (array) Product Images stores an array of images for the item, based on the available variants (SKUs)
```
Comment (Gidgidonihah) [inline _includes/product/response/item.md]: The wording here is awkward.

File: _includes/product/response/item.md [hunk 3]
```diff
@@ -65,4 +65,4 @@ custom_attributes
 : (object) The Custom Attributes object contains any special or custom-created attributes provided by the Supplier for this Item.
 
 categories
-: (list) The Categories list contains a list of the categories, as provided by the Supplier, for this Item.
+: (array) Categories contains an array of the categories, as provided by the Supplier, for this Item.
```
Comment (Gidgidonihah) [inline _includes/product/response/item.md]: All these that give the name of the field and says it contains an array are a bit awkward. The value of it IS an array, it doesn't CONTAIN an array.

Good: `<keyname> is an array of...`
Better: `An array of...`

This applies across the whole PR.

File: _includes/product/response/map_range.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 min
-: (number) The Minimum MAP for one of the SKUs or Item-variants
+: (integer) The Minimum MAP for one of the SKUs or Item-variants
```
Comment (Gidgidonihah) [inline _includes/product/response/map_range.md]: Wouldn't MAP (minimum advertised price) be a decimal?

File: _includes/product/response/map_range.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
 
 max
-: (number) The Maximum MAP for one of the SKUs or Item-variants
\ No newline at end of file
+: (integer) The Maximum MAP for one of the SKUs or Item-variants
```
Comment (Gidgidonihah) [inline _includes/product/response/map_range.md]: Wouldn't MAP be a decimal?

File: _includes/product/response/map_range.md (hunk 4 above)
Comment (nickcoleman) [reply _includes/product/response/map_range.md]: Yes

File: _includes/product/response/map_range.md (hunk 5 above)
Comment (nickcoleman) [reply _includes/product/response/map_range.md]: yes

File: _includes/product/response/item.md (hunk 3 above)
Comment (nickcoleman) [reply _includes/product/response/item.md]: 👍

**Pull request merged**